gnowsys-ndf/gnowsys_ndf/ndf/views/ajax_views.py

```diff
@@ -7086,6 +7088,26 @@ def export_to_epub(request, group_id, node_id):
         pass
     return HttpResponseRedirect(reverse('unit_detail', kwargs={'group_id': group_id}))
 
+
+
+def export_unit_to_epub(request,group_id):
+
+    from gnowsys_ndf.ndf.views.export_unit_to_epub import *
+    response_dict = {'success': False}
+    try:
+        group_obj = node_collection.one({'_id': ObjectId(group_id)})
+        epub_loc = create_epub(group_obj)
+        print 'hey!'
+        zip_file = open(epub_loc, 'rb')
+        response = HttpResponse(zip_file.read(), content_type="application/epub+zip")
+        response['Content-Disposition'] = 'attachment; filename="' + slugify(group_obj.name) + '.epub"'
+        return response
+    except Exception as export_fail:
+        print "\n export_fail: ", export_fail
+        pass
+    return HttpResponseRedirect(reverse('unit_detail', kwargs={'group_id': group_id}))
+
+
```

Comment:
Try to use the existing method and make it more versatile. Method behavior should be decided by args.